lib/handlers/map.put.js

```diff
@@ -66,7 +66,7 @@ const MapPut = class MapPut {
             const path = createFilePathToImportMap(incoming);
             const queue = [];
 
-            const busboy = new Busboy({
+            const busboy = Busboy({
```

Comment:
feels like this should be lower case busboy now that it's not a constructor?

Reply:
True. I'll fix that in a separate PR actually.